crowbar_framework/app/helpers/installer/upgrades_helper.rb

```diff
@@ -39,14 +57,6 @@ def restore_button
       button_tag t(".restore_button"), class: "btn btn-primary restore_button"
     end
```

Comment:
I guess this method can be completely removed if we really don't have the "restore" button anymore?